content/concept-keys.md

```diff
@@ -0,0 +1,84 @@
+    "master_seed": "PrivateKeyXXXXXXXXXXXXXXXXXXX",
+    "master_seed_hex": "PrivateKeyHexXXXXXXXXXXXXXXXXXXX",
+    "public_key": "aBQEK3r6hb8isnCWmortkNdv3MSMKP74o7gj3kHZQF9vDkbXo4Qs",
+    "public_key_hex": "032D911AE9887278110A8FE25169546D4E95A82AF5EFA25525FC3FDFFDC5A8B604"
+  },
+  "status": "success",
+  "type": "response"
+}
+```
+
+As you can see, in the context of the XRP Ledger, a key pair is composed of a bit more than a private key and a public key.
+
+
+#### Private Key
+
+The `master_key`, `master_seed`, and `master_seed_hex` are the private key in various formats, all of which can be used to sign transactions. Despite being prefixed with `master_`, these keys are not necessarily the master keys for an account. In this context, the `master_` prefix refers more to the keys' role as private keys.
+
+
+#### Public Key
+
+The `public_key` and `public_key_hex` are the public key in various formats, with the `public_key_hex` being the public key used to sign transactions.
+
+***TODO: Question: Okay, this is a nit of a clarification -- but I need to mention it to ensure that I understand how this works. When we talk about signing transactions - specifically authorizing transactions - from what I've seen of the `SetRegularKey` and `Payment` transactions, the sender must sign these transactions with a private key. This makes sense to me. In the paragraph above, we say that the `public_key_hex` is the "public key used to sign transactions." A sender cannot use a public key to sign a transaction, correct? Is what we are really trying to say that the `public_key_hex` value is the value that the XRP Ledger exposes publicly as the `SigningPubKey` to indicate the signer that authorized the transaction (the public-facing version of the private key?)***
+
+
+#### `account_id`
+
+The `account_id` is a [hash of the public key](concept-accounts.html#address-encoding) and designates the *potential* for an account to be created in the XRP Ledger. It is important to know that while an `account_id` exists, no actual account exists in the XRP Ledger until the `account_id` receives its first XRP payment. To create a funded account stored in the ledger, the `account_id` must [receive a `Payment` transaction](reference-transaction-format.html#creating-accounts) that provides enough XRP to meet the [reserve requirement](concept-reserves.html).
+
+For more information about the `wallet_propose` response, see [`wallet_propose`](reference-rippled.html#wallet-propose).
+
+You use this generated key pair in one of two ways: as master keys or as regular keys.
+
+
+## Master Keys
+
+Master keys have more powerful signing authority than regular keys. In addition to being able to sign all transactions that regular keys can, master keys are the only keys that can be used to perform the following actions:
+
+* Use the master private key to [disable the master public key](reference-transaction-format.html#accountset-flags).
```

Comment:
Got it about the parallel construction. 👍 Fixed. Just for the record (I don't want you to think you hired a tech writer who doesn't care about parallel construction - I do! 😸 ) -- I intentionally wrote "Use the master private key..." b/c the other half of the sentence is about the master public key. I thought it might be confusing to say, in effect: "Use the master keys to disable the master public key." But I think the bigger issue is needing to clarify what I mean by “master keys” and whether it is plural or not. I think other parts of the review answered these questions and I reworded this section -- and the structure is parallel now! Hopefully the content is also accurate.